docs/jobs-queue/queues.mdx

```diff
@@ -25,7 +25,33 @@ Then, you could configure two different runner strategies:
 
 ## Executing jobs
 
-As mentioned above, you can queue jobs, but the jobs won't run unless a worker picks up your jobs and runs them. This can be done in two ways:
+As mentioned above, you can queue jobs, but the jobs won't run unless a worker picks up your jobs and runs them. This can be done in four ways:
+
```

Comment:
I don't know why git is showing this diff here if the change in this file was made in the first PR:
https://github.com/payloadcms/payload/pull/10401/files